elements/pfe-cta/src/pfe-cta.html

```diff
@@ -1,4 +1,5 @@
 <span class="pfe-cta--wrapper">
     <slot></slot>
```

Comment:
If we add `&nbsp;` between the slot and the svg and both are set to `display: inline`, this might fix our wrapping issue.

Reply:
We'll need to adjust the padding to compensate though.